auto.js

```diff
@@ -488,6 +492,8 @@ var CookieAuto = {};
 
                 this.shoppingListSortMode = this.sorting.price;
 
+                this.ui.init.call(this.ui);
```

Comment:
Another instance of using call where it doesn't do anything more than just calling the function directly

Reply:
done